org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/testexecution/TestExecutorResource.xtend

```diff
@@ -58,6 +58,13 @@ class TestExecutorResource {
 		}
 	}
 
+	@GET
+	@Path("status/all")
+	@Produces(MediaType.APPLICATION_JSON)
+	def Iterable<TestStatusInfo> getStatusAll(@QueryParam("resource") String resourcePath) {
```

Comment:
Fair question. Nothing at all :disappointed:
Must be a copy/paste thing, and I never noticed it because there is no unit test (all this method does is wrap the corresponding method of `TestStatusMapper`, which _is_ unit-tested, and the integration test never complains, because Jersey simply passes in `null` when the query parameter is not present on the request URL…)